static/app/data/platformPickerCategories.tsx

```diff
@@ -202,16 +201,10 @@ const categoryList = [
     id: 'gaming',
     name: t('Gaming'),
     platforms: gaming,
-    display: (organization?: Organization) =>
-      organization?.features.includes('project-creation-games-tab') ?? false,
   },
   {
     id: 'all',
     name: t('All'),
     platforms: createablePlatforms,
   },
 ];
```

Comment:
For my own understanding: does `const` mean we cannot assign another value to `categoryList` or does it also mean that the list is immutable?

Not sure if it is good practice to expose a mutable list in JS.

Reply:
In JS, const prevents reassignment of the variable itself, but it does not make the value immutable. The contents of objects or arrays can still be modified.

For example:

```
// this is not allowed
categoryList = [4, 5, 6];
```

```
// this is allowed 
categoryList.push(4);   
categoryList[0] = 99;
categoryList.pop();
```